pkg/rke/k8s_rke_system_images.go

```diff
@@ -12072,6 +12072,46 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			WindowsPodInfraContainer:  "rancher/mirrored-pause:3.7",
 			Nodelocal:                 "rancher/mirrored-k8s-dns-node-cache:1.23.0",
 		},
+		// Enabled in Rancher v2.11.5 as part of RKE1 extended support
```

Comment:
We're not going to be embedding these RKE1 versions into any officially released Rancher versions as these changes won't get merged to `release/v2.11`. So best to just mention OOB release.
```suggestion
		// Enabled out of band as part of RKE1 extended support
```